modules/utxo-coredao/test/unit/utils.ts

```diff
@@ -0,0 +1,23 @@
+import { promises as fs } from 'fs';
+import { Descriptor, Psbt } from '@bitgo/wasm-miniscript';
+import * as utxolib from '@bitgo/utxo-lib';
+
+export async function getFixtureString(path: string, defaultValue: string): Promise<string> {
```

Comment:
little quality of life improvements

I should factor this into a library some day...

```suggestion
export async function getFixture(path: string, defaultValue: unknown): Promise<unknown> {
```